releasenotes/notes/fix-asm-request-context-partial-flushing-7c45aaf55e836261.yaml

```
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    appsec: Fixes encoding error when using AppSec and a trace is partial flushed.
```

nit, we're already pretty inconsistent but according to the docs the correct scope is ASM.

```suggestion
    ASM: Fixes encoding error when using AppSec and a trace is partial flushed.
```

---

damn, I thought I saw `appsec` being used and assumed it was the better one 🤦🏻 thanks for catching